en/docs/guides/user-mgt/lock-account.md

```diff
@@ -8,6 +8,67 @@ Account locking and account disabling are security features in WSO2 Identity Ser
 
 ---
 
+## Lock user accounts using SCIM
+
+1. Open the `<IS-HOME>/repository/conf/scim2-schema-extension.config` file and add the following configuration. 
+
+   ```
+   {
+      "attributeURI":"urn:ietf:params:scim:schemas:extension:enterprise:2.0:User:accountLock",
+      "attributeName":"accountLock",
+      "dataType":"boolean",
+      "multiValued":"false",
+      "description":"Lock user account.",
+      "required":"false",
+      "caseExact":"false",
+      "mutability":"readwrite",
+      "returned":"default",
+      "uniqueness":"none",
+      "subAttributes":"null",
+      "canonicalValues":[],
+      "referenceTypes":[]
+   }
+   ```
+
+2. Then add the `accountLock` attribute as a sub-attribute of User.
+
+   ```
+   "subAttributes":"verifyEmail askPassword accountLock employeeNumber costCenter organization division department manager"
+   ```
+
+3. Save the file and restart the server. 
+
+Before locking/unlocking users using SCIM, you need to do the following. 
+
+1. [Add claim mapping](insert-link)
+
+2. [Enable account locking](insert-link)
+
+### Test it Out 
+
+1. In order to update the lock status of a user account, we need to obtain the SCIM ID of that particular user. Therefore, we first call the GET users API to get the user details.
+
+   **Request**
+
+   ``` curl 
+   curl -v -k --user admin:admin 'https://localhost:9443/scim2/Users'
+   ```
+   **Sample**
+
+   ```curl
+   curl -v -k --user admin:admin 'https://localhost:9443/scim2/Users?filter=userName+Eq+vihanga'
+   ```
```

Comment:
code tabs?

Reply:
These do not have responses yet. Will include code blocks after there is a sample response too.